src/main/java/codes/theotter/kixstar/listener/BlockListener.java

```diff
@@ -56,7 +56,7 @@ public void onBlockInteraction(PlayerInteractEvent event) {
         }
 
         SafeTTeleporter teleporter = KixstarWorldSigns.getInstance().getMultiverseCore().getSafeTTeleporter();
-        MVDestination destination = KixstarWorldSigns.getInstance().getMultiverseCore().getDestFactory().getDestination(sign_block.getLine(1).trim());
+        MVDestination destination = KixstarWorldSigns.getInstance().getMultiverseCore().getDestFactory().getDestination("w:" + sign_block.getLine(2).trim());
```

Comment:
While the idea is great, it also doesn't take the core functionality in mind that the permissions are "optional". Only "named" signs are permission locked.

Reply:
In that case we will need to restrict who can place the signs, which would also mean we'd need to restrict who can destroy them (or destroy them with WorldEdit, for instance). Else people can just set the permission to be a different one and teleport anyways.

For now I think it's easiest to give people permission explicitly. In the case of the default `world`, the permission `kixstarworlds.world.world` overrides the config check.

Reply:
Sure